test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -69,3 +79,53 @@ def edit_community_channel(self, communityChannelName: str, newCommunityChannelN
         type(CreateOrEditCommunityChannelPopup.COMMUNITY_CHANNEL_NAME_INPUT.value, newCommunityChannelName)
         click_obj_by_name(CreateOrEditCommunityChannelPopup.COMMUNITY_CHANNEL_BUTTON.value)
         time.sleep(0.5)
+
+    def _select_test_image(self, fixtures_root: str, image_index: int):
```

Comment:
I agree, do note though that for all the actions inside the file dialog, I cannot do anything, because I can't target objects there, it falls out of the app.

I'll try to do it for going back to the app though